routes/pictures.js

```diff
@@ -40,21 +44,19 @@ router.post('/', async (req, res) => {
 router.put('/:picturesId', async (req, res) => {
   try {
     const { picturesID } = req.params;
-    const { id, furnitureId, imageURL } = req.body;
+    const { donationId, imageUrl, notes } = req.body;
     const updatedPictures = await db.query(
-      `UPDATE pictures (
-        id, furnitureId, imageURL,
-        )
+      `UPDATE pictures
       SET
-        id = $(id),
-        furnitureId = $(furnitureId),
-        imageURL = $(imageURL)
+        donation_id = $(donationId),
+        ${notes ? 'notes = $(notes), ' : ''}
+        image_url = $(imageUrl)
       WHERE id = $(picturesID)
       RETURNING *;`,
       {
-        id,
-        furnitureId,
-        imageURL,
+        donationId,
+        imageUrl,
+        notes,
         picturesID,
       },
```

Comment:
for `put` routes all parameters should be optional, also do an edge case check if nothing is passed in